README.md

```diff
@@ -54,56 +55,51 @@ To see detailed help for the entity extraction command, type:
 (venv) user@host:~$ ratom entities -h
 ```
 
-To run the extractor with default settings over a PST file or directory of PST files, type the following:
+To run the extractor with default settings over a PST or mbox file, or a directory containing one or more PST and mbox files, type the following:
 
 ```shell
-(venv) user@host:~$ ratom entities -p /path/to/PST-file-or-directory
+(venv) user@host:~$ ratom entities -p /path/to/PST-or-mbox-file-or-directory
 ```
 
 Progress is displayed in a bar at the bottom of the window. To terminate a job early and shut down all workers, type Ctrl-C.
 
 By default, the tool will use the spaCy en\_core\_web\_sm model, and will start as many concurrent jobs as there are virtual cores available. Entities are written to a sqlite3 file automatically named using the existing file or directory name and current datetime stamp, and with the following schema:
 
+![RATOM database schema](https://libratom.github.io/ratom-db-schema.svg)
+
+The schema contains 3 tables representing file information, message information and entity information.
+
+In the entity table, text is the entity instance, label\_ is the entity type, filepath is the PST or mbox file associated with this entity. Full message and file information for each entity are also available through message_id and file_report_id respectively. Note that pff_identifier (a message ID specific to PST files) will not be populated for messages located in mbox files. Examples of how to query these tables can be found in the **Interactive examples** section near the end of this README.
+
+## Advanced CLI uses
+
+The CLI provides additional flags to tune performance, output location, and verbosity of the tool. Some example use case are provided below.
+
+To use a different entity model, use the --space-model flag. The following example directs the tool to use the multi-language model:
+
 ```shell
-sqlite> .schema
-CREATE TABLE file_report (
-	id INTEGER NOT NULL,
-	path VARCHAR,
-	name VARCHAR,
-	size INTEGER,
-	md5 VARCHAR,
-	sha256 VARCHAR,
-	PRIMARY KEY (id)
-);
-CREATE TABLE message (
-	id INTEGER NOT NULL,
-	pff_identifier INTEGER,
-	processing_start_time DATETIME,
-	processing_end_time DATETIME,
-	file_report_id INTEGER,
-	PRIMARY KEY (id),
-	FOREIGN KEY(file_report_id) REFERENCES file_report (id)
-);
-CREATE TABLE entity (
-	id INTEGER NOT NULL,
-	text VARCHAR,
-	label_ VARCHAR,
-	filepath VARCHAR,
-	message_id INTEGER,
-	file_report_id INTEGER,
-	PRIMARY KEY (id),
-	FOREIGN KEY(message_id) REFERENCES message (id),
-	FOREIGN KEY(file_report_id) REFERENCES file_report (id)
-);
+(venv) user@host:~$ ratom entities -p --spacy-model xx_ent_wiki_sm /path/to/PST-or-mbox-file-or-directory
 ```
 
-The schema contains 3 tables representing file information, message information and entity information.
+To specify the number of jobs that may be run concurrently, use the -j flag. The following example limits the number of concurrent jobs to 2:
```

Comment:
Nit: It might be more accurate to write `The following example sets the number of concurrent jobs to 2:` instead of `limits`. I believe specifying jobs=n will result in n processes regardless of how many CPU cores the system has.

Reply:
Fixed.